xikolo-tvos/ViewControllers/CourseListViewController.swift

```diff
@@ -19,19 +19,6 @@ class CourseListViewController : AbstractCourseListViewController {
         resultsControllerDelegateImplementation.headerReuseIdentifier = "CourseHeaderView"
     }
 
-    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
```

Comment:
XCode somehow "automatically" showed me the tvOS-file next to the iOS-storyboard. 😠